### ISSUE TITLE
Incorrect exercise: [courseId=grammar-intro_eng][lessonNumber=12][exerciseId=3466]

### ISSUE THREAD

**Issue by ZacharyRMcKee** [jqueiroz/lojban.io#61]
# Description
user should not be marked incorrect for specifying tenses used in the prompt 

# Diagnostics information (do not remove)
- Exercise identifier: [courseId=grammar-intro_eng][lessonNumber=12][exerciseId=3466]
- Exercise payload: {"sentences":[{"text":"The future buyer is happy.","lojbanic":false}],"title":"Translate this sentence","type":"typing"}
- Attempted solution: {"text":"lo ba te vecnu ku ca gleki"}
- Attempted solution was classified as correct: no
- Validation payload: {"canonicalAnswer":"lo ba te vecnu cu gleki","correct":false}

**Comment by jqueiroz** [comment]
Agree. Unfortunately, this is difficult to implement witthout negatively impacting things like "lo pu vecnu" as well (where I do want tenses to be mandatory).

I will add a remark in the lesson about tenses that, unless explicitly requested in the prompt (e.g. "Translate specifying tenses"), tenses should never be specified.
